lib/chef/knife/ec_backup.rb

```diff
@@ -86,15 +86,17 @@ def download_user_acl(username)
         end
       end
 
-      def export_users_from_sql
+      def export_from_sql
         require 'chef/knife/ec_key_export'
         Chef::Knife::EcKeyExport.deps
         k = Chef::Knife::EcKeyExport.new
-        k.name_args = ["#{dest_dir}/key_dump.json"]
+        k.name_args = ["#{dest_dir}/key_dump.json", "#{dest_dir}/key_table_dump.json"]
         k.config[:sql_host] = config[:sql_host]
         k.config[:sql_port] = config[:sql_port]
         k.config[:sql_user] = config[:sql_user]
         k.config[:sql_password] = config[:sql_password]
+        k.config[:skip_users_table] = !config[:with_user_sql]
+        k.config[:skip_keys_table] = !config[:with_key_sql]
```

Comment:
Yes, namely restoring to an EC 11.x server that doesn't have the keys table.  However, perhaps it makes sense to have a single option that does the right thing based on what exists in the backup and what server version we are talking to.